guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -92,11 +92,13 @@ ifdef::satellite[]
 # yum clean metadata
 ----
 +
-. Ensure that the `rubygem-foreman_maintain` package that provides `{foreman-maintain}` is installed and up to date:
+. The `rubygem-foreman_maintain` is installed from the {Project} {ProjectVersion} maintenance repository or upgraded from the {Project} {ProjectVersion} maintenance repository if currently installed.
++
+Ensure {SmartProxy} has access to repository `{RHEL}-7-server-satellite-maintenance-{ProjectVersion}-rpms` and execute:
```

Comment:
can you check if an attribute exists rather than introducing a repo name when all others are handled with attributes.

Reply:
The section for **Upgrading a Connected Satellite Server** has it listed as `{Project} Maintenance repository`. No specific attribute exists for **Satellite Maintenance repository**. Changing it to that for consistency.

Reply:
Thank you!

Reply:
No problem! Let me know if you think it's good to be merged. :)